packages/lesswrong/components/hooks/useDismissable.ts

```diff
@@ -0,0 +1,33 @@
+import { useCallback, useState } from "react";
+import { useTracking } from "../../lib/analyticsEvents";
+import { useCookiesWithConsent } from "./useCookiesWithConsent";
+import moment from "moment";
+
+export const useDismissable = (
```

Comment:
I know we recently spoke about maybe using local storage instead of cookies for this, but it seems like the idea/infrastructure isn't quite ready yet. I figured I'd at least extract the logic into a hook, then we can just update it here in the future if we want to switch.

Reply:
I think in this particular case we want to use cookies rather than local storage right? Since the new post page is only accessible for logged in users.